src/xdsl/translation.py

```diff
@@ -0,0 +1,126 @@
+from xdsl.ir import *
+from xdsl.parser import *
+from xdsl.printer import *
+from xdsl.dialects.irdl import *
+from xdsl.dialects.builtin import *
+from xdsl.xdsl_opt_main import *
+from xdsl.printer import Printer
+# from xdsl.mlir_converter import MLIRConverter
+
+
+class Translation:
```

Comment:
I think this would also benefit from a stream field, where one can decide where to print it, similar to how the printer has. I understand though, that this PR is only about the introduction of this translator, so adding functionality might be a concern for later.